apis/apis.json

```diff
@@ -621,7 +621,7 @@
 
   {
     "id": "Google.Cloud.Spanner.Admin.Database.V1",
-    "targetFrameworks": "netstandard2.0",
+    "targetFrameworks": "netstandard2.0;net45",
     "testTargetFrameworks": "netcoreapp2.0;net461",
```

Comment:
After bringing net45 back shouldn't tests run against net45 as well?

Reply:
Testing net461 will run the tests on the .NET framework - and IIRC, even specifying net45 here would still run under "the latest .NET framework installed" due to the way installations work.